formats/application/vnd/gs1/produce/receiving_event/1/+json/schema.js

```diff
@@ -0,0 +1,43 @@
+var schemaUtil = require('../../../../../../../../lib/schema-util');
+var      vocab = require('../../../../../../../../vocabs/gs1');
+
+var restrictItemsTo = schemaUtil.restrictItemsTo;
+var vocabTermsToSchema = schemaUtil.vocabTermsToSchema;
+var requireValue = schemaUtil.requireValue;
+
+module.exports = schemaUtil.oadaSchema({ //reusing OADA schema validation logic
+  description:  
+  
+'GS1 receiving event has certain required fields and and array element',
```

Comment:
What does GS1 use as a description of what a "Receiving event" is?  This description doesn't have much to say to non-technical domain people.